### ISSUE TITLE
Updating Signal K Server then Restart does not apply update

### ISSUE THREAD

**Issue by LinnJS** [SignalK/signalk-server#1699]
## Description
Hello Signal K team love the new bug fixes in the 2.6.0 & 2.6.1 updates. I have tried to upgrade Signal K Server via admin dashboard several times. I have tried updating the server and restart and the update does not apply and stays on version 2.4.1. 

How might I fix this issue? Is there a way to manually upgrade the server on the Pi? Just dont want to break all my current configurations and plugins.

Thanks in advance 😄

**Comment by tkurki** [comment]
Have checked the [FAQ](https://github.com/SignalK/signalk-server/wiki/FAQ:-Frequently-Asked-Questions#server-update-not-updating-the-server-version)?

**Comment by LinnJS** [comment]
@tkurki thank you for the super quick reply. I am looking at the FAQ now and that seems to be the exact issue. Sorry did not see the FAQ when I was googling around for a fix.

When I cat the signalk-server file I see its using `/usr/local/lib/node_modules/signalk-server/bin/signalk-server` which seems to be the correct node_modules location for the new version. Although when looking at the node_modules paths I did see the old location. Is there a different place I should be looking to swap over the location? Or is there something I am missing here?

**Comment by tkurki** [comment]
Is this still open?

**Comment by LinnJS** [comment]
Issue resolved, thank you.

**Comment by LinnJS** [comment]
I would hate to re-open this issue after just closing it. But I went deep tonight on this issue and did a fresh install of the signalk-server and upon the fresh install it defaulted to the older version I am currently on and when trying to upgrade the server I got install failed. I am not sure how to troubleshoot this.

I removed all old versions of the server in the other location as the FAQ suggest and its all cleaned up. I defaulted back to the old config luckily it works but the data explorer now shows no data coming from the NMEA system and I am not sure how to troubleshoot this. I can get access to the boat but I am not sure how to begin troubleshooting data not getting into the system it was working before I started upgrading plugins not sure if the updated plugins would cause issues of the version server I am on which is version 2.4.1.

I am using node v18.17.0 and npm v9.6.7.

Also when I did the fresh install of the signalk-server I tried to upload the recovery file and I got an error saying it does not start with signalk even though it was the exact file I downloaded from the admin dashboard.
